config/config.c

```diff
@@ -12,42 +12,43 @@ bool init_config()
 {
     FILE *config_file = fopen(config_path, "rb");
 
-    if (config_file == NULL || !fread(&config, sizeof(struct config), 1, config_file))
+    if ( config_file == NULL || !fread(&config, sizeof(struct config), 1, config_file) )
     {
         config.aimbot.key.is_pressed = false;
 
         config.misc.bunny_hop = 1;
-        config.misc.autostrafe = 1;
+        config.misc.legit_autostrafe = 0;
+        config.misc.rage_autostrafe = 1;
         config.misc.do_thirdperson = false;
         config.aimbot.aimbot_enabled = 1;
         config.aimbot.draw_fov = 1;
         config.aimbot.fov = 10.0f;
-        config.aimbot.fov_color = (struct nk_colorf){ 207.0f / 255.0f, 115.0f / 255.0f, 54.0f / 255.0f, 0.25f };
+        config.aimbot.fov_color = (struct nk_colorf) {207.0f / 255.0f, 115.0f / 255.0f, 54.0f / 255.0f, 0.25f};
```

Comment:
could you please provide style guidelines, i tried my best to replicate it on my .clang-format and i messed this up